arangod/Cluster/ClusterMethods.cpp

```diff
@@ -1245,7 +1245,12 @@ futures::Future<OperationResult> countOnCoordinator(transaction::Methods& trx,
   auto* pool = trx.vocbase().server().getFeature<NetworkFeature>().pool();
   for (auto const& p : *shardIds) {
     network::Headers headers;
-    ClusterTrxMethods::addTransactionHeader(trx, /*leader*/ p.second[0], headers);
+    if (p.second.empty()) {
```

Comment:
```suggestion
    TRI_ASSERT(!p.second.empty());
    if (p.second.empty()) {
```
I agree with the below error handling and like it.
However I think we should never actually end up in this situation.
I would like to get this smashed into my face on Jenkins.
And not by some circumstances silently ignored.